crates/pathfinder/src/core.rs

```diff
@@ -416,10 +416,8 @@ impl Chain {
         match self {
             // SN_MAIN
             Chain::Mainnet => StarkHash::from_u128(0x534e5f4d41494eu128),
-            // SN_GOERLI
-            Chain::Testnet => StarkHash::from_u128(0x534e5f474f45524c49u128),
-            // SN_GOERLI2
-            Chain::Testnet2 => StarkHash::from_u128(0x534e5f474f45524c4932),
+            // SN_GOERLI or SN_GOERLI2
```

Comment:
```suggestion
            // Both testnets have the same ID: SN_GOERLI
```
Think having a comment, just to make sure its understood that this is on purpose like this.